.husky/pre-commit

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+if npm list prettier | grep -q prettier@ ; then
```

Comment:
Is this going to run on windows?

Reply:
It looks like a shell script? Does husky somehow run shell scripts on windows?

Reply:
I'm running it on windows, the vanilla pre-commit script through git is able to run the same way, so it's probably using git bash